README.md

```diff
@@ -126,7 +123,7 @@ await reactiveBleClient.requestConnectionPriority(deviceId: 'AA:BB:CC:DD:EE:FF',
 #### Clear GATT cache
 The Android OS maintains a table per device of the discovered service in cache. Sometimes it happens that after a firmware update a new service is introduced but the cache is not updated. To invalidate the cache you can use the cleargattCache operation. 
 
-**This is a hidden BLE operation and should be used with extreme caution since usage is not encouraged by the Android team**  
+** This is a hidden BLE operation and should be used with extreme caution since this operation is on the greylist https://developer.android.com/distribute/best-practices/develop/restrictions-non-sdk-interfaces . API is still working on Android Q  **  
```

Comment:
I don't think spaces are needed `**in between of asterisks**`. Shall we make a nice Markdown link out of the URL?